### DISCUSSION TITLE
Syndicate Wars on CDs from game magazines

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#117] [General]
Here's some info on where to find SW-related materials in various game mags:

[Executable catalogue of upcoming games](https://tcrf.net/Prerelease:Syndicate_Wars_(DOS)#Press_Kit_Images):
```
Score 1995-08_20 cd
PC Player DE 06/1995 CD
```

The Bullfrog Progress Report 1995 video:
```
OK PC magazine (ESP) 42 (1996.01) CD1
```

The [Bullfrog Christmas CD](https://tcrf.net/Proto:Syndicate_Wars_(DOS)#Pre-Alpha_Demo):
```
Level 1996-01 cd
Score 1996-01 25 cd (with binary modification of HDD folder created)
PC Zone 37 (April 1996) CDZone
PC-Action 1996-03 (Creation demo only)
OK PC magazine (ESP) 42 (1996.01) CD2 (older version, with "Sony" logo)
SBG (russian CD mag) 1996/5 (SW demo and DK video)
```

The [two-level SW demo](https://tcrf.net/Proto:Syndicate_Wars_(DOS)#Adrenaline_Vault_Exclusive_Demo):
```
PCGAMER Disc 2.12 JAN 1997
PC Games dot exe 1996-12 CD
```

The CDs with the [standard pre-release SW demo](https://tcrf.net/Proto:Syndicate_Wars_(DOS)#Pre-Release_Demo):
```
Level 1996-10 cd
Micromania Tercera Epoca ESP CD 21 (incl. music WAV file)
PC Format 61 - Oct 1996
Personal Computer World nov96 (disk no 3)
Score 1996-11_35 cd
Secret Service PL cover CD_40
CD Action PL 1996-11 CD_B (incl. music WAV file)
PC-Action DE 1996-10 CD
PC Gamer UK #036 CD
PC Player DE 10/1996 CD
PC Review #61 CD_Review
Excalibur Magazine CZ CD 57 1996-06 (incl. music WAV file)
GAMBLER CD 02 1996/12
Joystick #74 FR cd
PC Master GRK cd11 nov1996
PC Powerplay AU CD #8
PC Pro UK 1996/12
SBG RU 1996/07
101 OnLy ThE BeSt GaMeS #7
```

CDs with post-release SW demo:
```
PC Gamer 2.12 - January 1997 Demo Disc
```

List of articles about the game:
https://www.abandonware-france.org/ltf_abandon/ltf_jeu.php?id=247&fic=medias

**Comment by Moburma** [comment]
Mike Diskett wrote an official making of column for GamesMaster magazine in 1996, scans of this below:

https://ibb.co/x6NX18g
https://ibb.co/NV5BXBp

https://ibb.co/2gWTv5W
https://ibb.co/9pdNrCg

https://ibb.co/Rz0LgDL
https://ibb.co/jHMZWbf

https://ibb.co/9nLybcn
https://ibb.co/6Fxgcbg

https://ibb.co/wpgrKyc
https://ibb.co/GFdmRsD

**Comment by Moburma** [comment]
Here's one I've not seen posted before, one of the very few anecdotes about making the game, this time from Retro Gamer issue 110:

![](https://i.ibb.co/CKtVtvG/retrogamer110.jpg)
